src/steamship/invocable/lambda_handler.py

```diff
@@ -27,194 +28,244 @@ def encode_exception(obj):
+
+    result = response.dict(by_alias=True, exclude={"client"})
+    # When created with data > 4MB, data is uploaded to a bucket.
+    # This is a very ugly way to get the deep size of this object
+    data = json.dumps(result.get("data", None)).encode("UTF-8")
+    data_size = sys.getsizeof(data)
+    logging.info(f"Response data size {data_size}")
+    if data_size > 4e6 and invocation_context.invocable_type == "plugin":
+        logging.info("Response data size >4MB, must upload to bucket")
+
+        filepath = str(uuid.uuid4())
+        signed_url = (
+            client.get_workspace()
+            .create_signed_url(
+                SignedUrl.Request(
+                    bucket=SignedUrl.Bucket.PLUGIN_DATA,
+                    filepath=filepath,
+                    operation=SignedUrl.Operation.WRITE,
                 )
-                .signed_url
             )
+            .signed_url
+        )
+
+        logging.info(f"Got signed url for writing: {signed_url}")
+
+        upload_to_signed_url(signed_url, data)
+
+        # Now remove raw data and replace with bucket
+        del result["data"]
+        result["dataBucket"] = SignedUrl.Bucket.PLUGIN_DATA.value
+        result["dataFilepath"] = filepath
+
+    if logging_handler is not None:
+        logging_handler.close()
+
+    logging.info(result)
+    return result
+
+
+def create_handler(invocable_cls: Type[Invocable]):
+    """Legacy wrapper function for a Steamship invocable within an AWS Lambda function. Called by code within a
+    plugin or package.
+    """
+    logging.info("Using legacy (unsafe imports) create_handler")
+    invocable_cls_lambda = lambda: invocable_cls  # noqa: E731
+    bound_internal_handler = lambda event, client: internal_handler(  # noqa: E731
+        invocable_cls_lambda, event, client
+    )
+    return lambda event, context=None: handler(bound_internal_handler, event, context)
+
 
-            logging.info(f"Got signed url for writing: {signed_url}")
+def safely_find_invocable_class() -> Type[Invocable]:
+    """
+    Safely find the invocable class within invocable code.
+    """
+    try:
+        import api
+    except Exception as e:
+        logging.exception(e)
+        raise SteamshipError(
+            message=f"There was an error loading the main file:\n{traceback.format_exc()}", error=e
+        )
 
-            upload_to_signed_url(signed_url, data)
+    invocable_classes = []
+    for element in [getattr(api, x) for x in dir(api)]:
+        if inspect.isclass(element):
+            # Using names and not issubclass(element, Invocable) because latter was returning false?
+            superclass_names = [c.__name__ for c in inspect.getmro(element)]
+            if "Invocable" in superclass_names and element.__module__ == "api":
+                invocable_classes.append(element)
+    if len(invocable_classes) == 0:
+        raise SteamshipError(message="Could not find package or plugin class in api.py.")
```

Comment:
Would this help the user to debug without reading the docs?

```suggestion
        raise SteamshipError(message="Could not find package or plugin class in api.py. Define your package or plugin by subclassing from PluginService or PackageService.")
```

Reply:
Yes, great suggestion.  I just slipped it into my next PR.